pkg/webhook/resources/restore/validator.go

```diff
@@ -92,6 +96,12 @@ func (v *restoreValidator) Create(request *types.Request, newObj runtime.Object)
 		return werror.NewInvalidError(fmt.Sprintf("please stop the VM %q before doing a restore", vm.Name), fieldTargetName)
 	}
 
+	if result, err := util.HasInProgressingVMRestoreOnSameTarget(v.vmRestore, vm.Namespace, vm.Name); err != nil {
+		return werror.NewInternalError(fmt.Sprintf("can't check whether there is another in progressing vmrestore, err: %+v", err))
```

Comment:
```suggestion
		return werror.NewInternalError(fmt.Sprintf("failed to get the VM-related restores, err: %+v", err))
```

Reply:
Fixed it in https://github.com/harvester/harvester/pull/2601. Thank you.